client/src/components/advancedSelectWidget/AdvancedSelect.js

```diff
@@ -313,11 +289,9 @@ export default class AdvancedSelect extends Component {
     })
   }
 
+  handleInteraction = showOverlay => this.setState({ showOverlay })
+
   handleHideOverlay = () => {
-    if (this.state.inputFocused) {
```

Comment:
Probably because of this change the following happens:
- I click on the attendees input field
- the overlay opens
- I select an attendee
- I click on the input field again in order to add a text filter
- but I can't because the overlay closes now